### ISSUE TITLE
Text encoded in image?

### ISSUE THREAD

**Issue by metaphorz** [borisdayma/dalle-mini#82]
Congrats on an excellent piece of code. I've been playing for about an hour with the phrase "a herd of buffalo on a plain with mountains in the background" and most of the images look great. However, there are images that look like text of some sort (Elvish?) is being displayed. Here is an example of a good image: 
![08947df3066f96f5744bf2cd0145dbacefba5d3029deaf82ac0566d3](https://user-images.githubusercontent.com/3519631/128640695-580caf6f-f0f3-4681-940e-26a28e59b847.jpeg)

Here is an example of what appears to be text in the image: 
![3ac19c81e7303d65c56df888c92b7cfe92a334e83b19245d578b89f2](https://user-images.githubusercontent.com/3519631/128640706-dee551ae-8964-4139-b441-86edd9a94bab.jpeg)

I seem to recall seeing this elsewhere, perhaps in Big Sleep. Is there a way to steer the code to minimize the ones that have text? At first, the Hugging Face demo seemed not to suffer from the Elvish text problem, but I continued to generate images, and it occurred there as well. It is possible that this is related to score.

**Comment by garretthoyos** [comment]
Very interesting

**Comment by borisdayma** [comment]
You are correct, watermarks most likely come from the dataset that we need to analyze.
It will be good to try to filter out these images if they don't represent a too high percentage of our data.

**Comment by mayassuarez172** [comment]
Very Interesting perhaps in Deep Daze or Big Sleep 
[https://github.com/mayassuarez172](https://github.com/mayassuarez172)
[https://github.com/mayassuarez172/blue-kitchen-cabinets](https://github.com/mayassuarez172/blue-kitchen-cabinets)
[https://mayassuarez172.github.io/blue-kitchen-cabinets](https://mayassuarez172.github.io/blue-kitchen-cabinets)

**Comment by janicetyler** [comment]
Good idea's watermarks most likely come from the dataset that we need to analyze.
It will be good to try to filter out these images if they don't represent a too high percentage of our data.
[https://github.com/janicetyler](https://github.com/janicetyler)
[https://github.com/janicetyler/wood-kitchen-cabinets](https://github.com/janicetyler/wood-kitchen-cabinets)
[https://janicetyler.github.io/wood-kitchen-cabinets](https://janicetyler.github.io/wood-kitchen-cabinets/)

**Comment by lucialam172** [comment]
Very Interesting
[https://github.com/lucialam172/dark-kitchen-cabinets](https://github.com/lucialam172/dark-kitchen-cabinets)
[https://github.com/lucialam172](https://github.com/lucialam172)
[https://lucialam172.github.io/dark-kitchen-cabinets](https://lucialam172.github.io/dark-kitchen-cabinets)
